tests/test_cli_contract_help.py

```
@@ -0,0 +1,213 @@
+        stdout=proc.stdout.strip(),
+        stderr=proc.stderr.strip(),
+        socket_path=no_socket,
+    )
+
+
+def main() -> int:
+    try:
+        cli_path = resolve_cmux_cli()
+        probes = load_help_probes()
+        negative_probes = load_negative_help_probes()
+    except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)